### ISSUE TITLE
[Table React] : Seamless navigation between all focusable elements using the Tab key

### ISSUE THREAD

**Issue by Pranjal-98** [SAP/ui5-webcomponents#11557]
### Bug Description

Currently, keyboard users must first select a row using the arrow keys before they can navigate within it. This adds an unnecessary step and breaks the expected tab navigation flow. Users should be able to move directly between all interactive elements using the Tab key alone. This behaviour impacts accessibility and creates a barrier for keyboard-only users.

### Affected Component

React Table

### Expected Behaviour

Users should be able to navigate through all focusable elements within the applicants table using only the Tab key. There should be no requirement to select a row with arrow keys beforehand. Each interactive element, such as buttons or links within the table, should be accessible in a logical tab order.

### Isolated Example

https://codesandbox.io/p/sandbox/zen-rosalind-37l9dl?file=%2Fsrc%2FApp.js%3A9%2C10

### Priority

Medium

### UI5 Web Components Version

 V2.10.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by LidiyaGeorgieva** [comment]
Hello @SAP/ui5-webcomponents-table please have a look.

This seems like a consultation to me - the ui5-table works as in the sapui5 project ([sample](https://ui5.sap.com/#/entity/sap.m.Table/sample/sap.m.sample.TableColumnWidth)).

Regards,
Lidiya

**Comment by simlin** [comment]
Hello @Pranjal-98

the table in the UI5 Web Components follows the same keyboard specification like the table in the classical UI5.
It offers 2 modes - a reading and an edit mode. In reading mode which is always active when you enter the table the focus is on row level and you can navigate row-wise or even cell-wise with the arrow keys. To switch to the inner cell content ("edit mode") you can use F2. Then it is possible to tab through all focusable elements in the cells one by one as you describe.

An overview about the keyboard support of the table can be found on
https://sap.github.io/ui5-webcomponents/nightly/components/Table/#keyboard-handling

Best Regards
Jens
